### ISSUE TITLE
Carthage build failing?

### ISSUE THREAD

**Issue by AndrewSB** [Thomvis/BrightFutures#93]
I'm trying to do a regular Carthage update, result fetches and builds fine, but then BrightFutures gives me:
```
*** Building scheme "BrightFutures-Mac" in BrightFutures.xcworkspace
** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
A shell task failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
```

**Comment by Thomvis** [comment]
Sorry to hear you're having problems

What version of Carthage are you using? The latest version worked for me.

**Comment by AndrewSB** [comment]
I'm on 0.9.4, their latest release too. Let me get verbose logs

**Comment by AndrewSB** [comment]
Ahh, I see. It's a problem with Result, I'll go talk to them :smile: 

```
*** Building scheme "BrightFutures-Mac" in BrightFutures.xcworkspace
Build settings from command line:
    ONLY_ACTIVE_ARCH = NO

=== BUILD TARGET Result-Mac OF PROJECT Result WITH CONFIGURATION Release ===

Check dependencies
target 'Result-Mac' has bitcode enabled (ENABLE_BITCODE = YES), but it is not supported for the 'macosx' platform

** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
A shell task failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
```

**Comment by AndrewSB** [comment]
It is an issue with BrightFutures, @Thomvis, https://github.com/antitypical/Result/issues/117
I'll try to make a PR later this evening

**Comment by ikesyo** [comment]
@Thomvis 

> What version of Carthage are you using? The latest version worked for me.

This is problematic only for Xcode 7.1, in which `ENABLE_BITCODE=YES` for Mac target is prohibited. That was OK for Xcode 7.0.

**Comment by Thomvis** [comment]
Thanks, both of you! I missed that it changed in Xcode 7.1 and will obviously need to update to the latest Result.

**Comment by AndrewSB** [comment]
merged in #94 :+1:

**Comment by firedev** [comment]
I am having Xcode 7.2 I believe and BrightFutures fails:

```
*** Building scheme "BrightFutures-Mac" in BrightFutures.xcworkspace
** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
A shell task failed with exit code 65:
** BUILD FAILED **


The following build commands failed:
	Check dependencies
(1 failure)
```

**Comment by Thomvis** [comment]
I'm seeing this as well. Any additional input is very welcome!

**Comment by Thomvis** [comment]
We'll have to wait until Result has published a release that is compatible with Xcode 7.2.

Related: You'll probably have to update to the newest version of Carthage in order to be able to build using Xcode 7.2.

**Comment by firedev** [comment]
Hi, sorry maybe I was using it wrong, this fixed it:

```
$ carthage bootstrap --use-submodules --platform ios
```

**Comment by Thomvis** [comment]
I've released 3.2.2, which should be fully compatible with Xcode 7.2. Also, don't forget to update Carthage to the latest version.
